converter/dicom-cast/docs/workingWithPrivateLink.md

```diff
@@ -0,0 +1,24 @@
+# Configuration steps for dicomast to work with privatelink enabled dicom service
+
+1. Create a virtual network with two subnets within the same subscription and region as you would plan to create the health data services workspace 
+![Alt text](image.png)
+
+2. Provision Healthdataservice workspace, dicomserver and fhir service in the same region.
+3. Enable Privatelink to Healthdataservice workspace. This privatelink would use the Virtual network created in step 1 and default subnet
+![Alt text](image-1.png)
+
+4. Use the template given [here](DicomcastDeploymentTemplate.md) to deploy dicomcast within a virtual network created in step 1. This will use the subnet that is delegated to Microsoft.containerinstance/containergroups as shown in picture in step 1. 
+5. Add the following role assignments to the workspace  on the container instances SAMI. 
```

Comment:
might want to just write system assigned managed identity so its clearer.  Also did they need to enable it or was it already enabled by default>

Reply:
Template deployment will take care of it